TESTS/mbed_platform/FileHandle/main.cpp

```diff
@@ -353,7 +374,15 @@ void test_fprintf_fscanf()
     TEST_ASSERT_TRUE(TestFile<FS>::functionCalled(TestFile<FS>::fnWrite));
     TEST_ASSERT_TRUE(std::ferror(file) != 0);
     TEST_ASSERT_TRUE(fprintf_ret < 0);
-
+#else
```

Comment:
Add a comment why the previous code is commented out for __MICROLIB and that you will be writting exactly 2 chars instead to get the same behaviour.